lib/hanami/configuration/middleware.rb

```diff
@@ -11,7 +11,7 @@ def initialize
       end
 
       def use(middleware, *args, &block)
-        stack.push([middleware, *args, block].compact)
+        stack.push([middleware, args, block].compact)
```

Comment:
@timriley wasn't certain what the purpose behind the `.compact` was here so left it in for the moment, but I think it can be removed.